tests/security/ebpf/disable_unprivileged_ebpf.pm

```diff
@@ -56,6 +69,35 @@ sub run {
     # Reboot, verify the eBPF status: should be disabled unprivileged eBPF again
     $self->reboot_and_check('2');
 
+    # Set 'unprivileged_bpf_disabled' value to 1 by using 'sysctl'
+    validate_script_output('sysctl kernel.unprivileged_bpf_disabled=1', sub { m/kernel.unprivileged_bpf_disabled = 1/ });
+    $self->privileged_bpf_check;
+
+    # Reboot, verify the eBPF status: should be disabled unprivileged eBPF again
+    $self->reboot_and_check('2');
+
+    # Set 'unprivileged_bpf_disabled' value to 1 by using "$f_unpriv_bpf_disabled"
+    assert_script_run("echo -n 1 > $f_unpriv_bpf_disabled");
+    $self->privileged_bpf_check;
+
+    # Install gcc compiler
+    assert_script_run("zypper -n in gcc", timeout => 300);
+    # Switch to common user, download and compile the C test code
+    select_console 'user-console';
+    assert_script_run("cd /tmp");
+    assert_script_run("wget " . autoinst_url . "/data/ebpf/bpf_test.c");
+    assert_script_run("gcc -o bpf_test bpf_test.c");
+
+    # bpf system call failed with 'Operation not permitted' on unprivileged users
+    validate_script_output("./bpf_test", sub { m/Operation not permitted/ });
+    # bpf system call should be successful on privileged users
+    become_root;
+    assert_script_run("cd /tmp");
```

Comment:
For coding style compliance purpose please check the all the `"` used if they can be replaced by `'`.

Reply:
Fixed.
btw, I also changed the `Operation not permitted` to `BPF: Operation not permitted`,
and `Success` to `BPF: Success` to avoid confusion.